test/controllers/trusts.involved.controller.spec.ts

```diff
@@ -0,0 +1,141 @@
+jest.mock('../../src/middleware/authentication.middleware');
```

Comment:
can you also add 
`jest.mock("ioredis");`
if you exclude it, it will cause the test to not finish properly and it will get timed out (as it is waiting for a Redis connection)

Reply:
completed